src/app/shared/validationMessages.model.ts

```diff
@@ -142,6 +144,12 @@ export const validationMessages = {
     maxlength: 'Workflow Path is too long. (Max 256 characters.)',
     pattern: `Must begin with '/' and end with '*.cwl', '*.yml', '*.yaml', '*.config', or'*.wdl' ` + 'depending on the descriptor type.'
   },
+  service_path: {
```

Comment:
I don't think the service_path is ever editable, so I don't think you need this section.

Reply:
changed input to text and so removed all the service path stuff from form validation